firmware/controllers/algo/defaults/default_fuel.cpp

```diff
@@ -109,6 +109,7 @@ static void setDefaultFuelCutParameters() {
 	engineConfiguration->coastingFuelCutTps = 2;
 	engineConfiguration->coastingFuelCutMap = 30;
 	engineConfiguration->coastingFuelCutClt = 60;
+	engineConfiguration->disableFuelCutOnClutch = false;
```

Comment:
By the way, you don't have to set any defaults that have a bit value of 0. The configuration is zeroed before reset, so everything defaults to 0/false/etc.